zanata-adapter-xliff/src/main/java/org/zanata/adapter/xliff/XliffWriter.java

```diff
@@ -216,30 +221,12 @@ public static void writeFile(File file, Resource doc, String locale,
             // end xliff tag
             writer.writeEndDocument();
             writer.flush();
+            writer.close();
```

Comment:
I am using java 7's try with resource. It will automatically close in finally block